ffig/filters/capi_filter.py

```diff
@@ -51,7 +51,7 @@ def to_go(t):
         if t.pointee.kind == TypeKind.CHAR_S:
             return 'string'
         if t.pointee.kind == TypeKind.RECORD:
-            return 'unsafe.Pointer'
+            return t.pointee.name.replace('const', '')
```

Comment:
I think clang is decent enough not to leave trailing spaces on identifier names.
It's still a hack we should improve upon, but a cleaner hack than most.